Apps/APN/package.json

```diff
@@ -11,7 +11,8 @@
     "clean-install-ios": "./scripts/clean-run-ios.sh",
     "dev:update": "../../scripts/dev:update.sh",
     "postinstall": "pod update --project-directory=ios",
-    "preinstall": "npm run dev:update"
+    "preinstall": "npm run dev:update --legacy-peer-deps",
+    "ci:install": "npm run preinstall && npm ci --legacy-peer-deps && npm run bundle:ios"
```

Comment:
Have to run with `--legacy-peer-deps` due to outdated dependencies, which can be fixed in another ticket

Reply:
I think it is the clipboard library that's quite outdated ! I am also working on a similar ticket, merging two similar PRs with similar intention (upgrade) might cause conflicts. 🤔

Reply:
Thanks for the callout Aman. Without this change, we won't be able to compile APN app on CI, which means we won't be able to verify the changes for mentioned ticket. I believe the conflicts for clipboard dependency would only be in the lock file, and once they are resolved, we can remove `--legacy-peer-deps` flag, making the sample apps consistent. Resolving lock file conflicts shouldn't be an issue in my opinion. Do you see any other challenges with this?

Reply:
Please disregard my previous comment. I won’t encounter conflicts until I merge all my changes into `main`, which isn’t happening anytime soon. Your changes will be merged into `main`, and any conflicts may arise when the CDP feature branch is eventually merged into main, which is expected. 👍